configure

```diff
@@ -1331,6 +1331,7 @@ cat > "$quick_prefix/quick.rc" << EOF
 export QUICK_INSTALL=$quick_prefix
 export QUICK_BASIS=\$QUICK_INSTALL/basis
 export PATH=\$QUICK_INSTALL/bin:\$PATH
+export ASE_QUICK=$installer
```

Comment:
Kio, what are you trying to achieve here? The variable `$installer` is used in a loop and will contain the name of the executable that is built. After the loop it contains the last executable that was built (quick, quick.MPI, quick.cuda, or quick.cuda.MPI).

ASE should decide which executable to use and check if it is present in `$QUICK_INSTALL/bin`. In case of the MPI executables it will need to call them with `mpirun -np N ` prefix.